website/src/pages/Blog/Author/index.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { useEffect, useState } from "react";
+import useStyles from "./styles";
+import { useTranslation } from "react-i18next";
+import {
+  Avatar,
+  Breadcrumbs,
+  Container,
+  Grid,
+  Link,
+} from "@material-ui/core";
+import Footer from "../../../components/Footer";
+import { VIEW_PORT } from "../../../constants";
+import Sponsor from "../../../components/Sponsor";
+import { getAvatar } from "../../../utils/getAvatar";
+import { useViewport } from "../../../hooks/viewportWidth";
+import { pageCount } from "../../../utils/getPageCount";
+import SeoJson from "../../../resources/seo.json";
+import { currentOrigin } from "../../../utils/currentHost";
+import { Metadata } from "../../../components/Metadata";
+import { useAuthorName } from "../../../hooks/extractBlogPath";
+import { Pagination } from "@material-ui/lab";
+import BlogCard from "../../../components/BlogCard";
+import ErrorPage from "../../ErrorPage";
+
+const Blog: React.FC = () => {
+  const { t } = useTranslation();
+  const classes = useStyles();
+  const [jsonMdData, setJsonMdData] = useState<any>("");
+  const authorName = useAuthorName();
+  const itemsPerPage = 6;
+  const [page, setPage] = React.useState<number>(1);
+  const { width } = useViewport();
+  const mobileBreakpoint = VIEW_PORT.MOBILE_BREAKPOINT;
+  
+
+  const fetchBlogs = async () => {
+    const { default: blogs } = await import(`../../../posts.json`);
+    setJsonMdData(blogs);
+  };
+
+  useEffect(() => {
+    fetchBlogs();
+  });
+
+  const filteredAuthorData = (jsonMdData || []).filter(
+    (blog: any) => blog.author.toLowerCase().replace(/[^\w ]+/g,'').replace(/ +/g,'-') === authorName
+  );
+
+  const pagination = () => {
+    return (
+      <Pagination
+      count={
+       pageCount(filteredAuthorData)
+      }
+      page={page}
+      onChange={(_event, val) => val? setPage(val) : setPage(1)}
+      className={classes.pagination}
+    />
+    );
+  };
+
+  return (
+    <>
+     <Metadata title={SeoJson.pages.blog.title} description={SeoJson.pages.blog.description} url={`${currentOrigin}${SeoJson.pages.blog.url}`} image={`${currentOrigin}${SeoJson.pages.blog.image}`} isPost={false} type="Series"  />
```

Comment:
Let's please make this more readable.

Reply:
This is added in same format for all the pages cc @IsAmrish

Reply:
> Let's please make this more readable.

did you mean reformatting??